cloudconfig/containerinit/container_userdata.go

```diff
@@ -241,24 +241,31 @@ func shutdownInitCommands(initSystem, series string) ([]string, error) {
 		// keep clean logs for diagnosing issues / debugging.
 		"/var/log/cloud-init*.log",
 	}
+	paths := strings.Join(cleanupOnShutdown, " ")
+	removeCmd := fmt.Sprintf("/bin/rm -fr %s\n  ", paths)
+	shutdownCmd := "/sbin/shutdown -h now"
+	name := "juju-template-restart"
+	desc := "juju shutdown job"
```

Comment:
This is simply visually moving the commonly used vars above the vars specific to the two different execStart cases.